pkg/bmc/redfish.go

```diff
@@ -12,6 +12,7 @@ func init() {
 	registerFactory("redfish-virtualmedia", newRedfishVirtualMediaAccessDetails)
 	registerFactory("ilo5-virtualmedia", newRedfishVirtualMediaAccessDetails)
 	registerFactory("idrac-virtualmedia", newRedfishiDracVirtualMediaAccessDetails)
+	registerFactory("idrac+redfish", newRedfishiDracAccessDetails)
```

Comment:
Should be `idrac-redfish`.

Reply:
was in an earlier patchset, but changed it trying to figure out go was throwing weird errors. According to stepehen it seems to be a rebase issue.